website/docs/developer/deployment-guide/deployment-upgrading-orca.md

```diff
@@ -0,0 +1,102 @@
+
+**IMPORTANT:** By convention, ORCA minor version releases introduce breaking 
+changes (e.g., 1.13.x -> 1.14.0), so it is critical that the release notes 
+are viewed for migration steps. Carefully read each BREAKING CHANGES and 
+MIGRATION STEPS sections within the `CHANGELOG.md` file, following all steps, 
+starting with the oldest release after your currently installed release, and 
+progressing through them chronologically.
+
+To view the released module artifacts for each ORCA core version, see the 
+[ORCA Releases page](https://github.com/nasa/cumulus-orca/releases).
+
+## Migrating to a New Version
+
+When breaking changes have been introduced, the ORCA Team will publish 
+instructions on migrating from one version to another. Detailed release notes 
+with migration instructions (if any) for each release can be found on the 
+[ORCA Releases page](https://github.com/nasa/cumulus-orca/releases).
+
+1.	**Use consistent ORCA versions:** All Terraform modules must be updated 
+to the same ORCA version number (see Updating ORCA Version below). **Check 
+the CHANGELOG for deprecation/breaking change notices.**
+
+2.	**Follow all intervening steps:** When skipping over versions, **you must 
+perform all intervening migration steps.** For example, if going from version 
+1.1.0 to 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0. This is 
+critical because each release that contains migration steps provide 
+instructions *only* for migrating from the *immediately* previous release, 
+but you must follow *all* migration steps between your currently installed 
+release and *every release* through the release that you wish to migrate to.
+
+3.	**Migrate lower environments first:** Migrate your "lowest" environment 
+first and test it to ensure correctness before performing migration steps in 
+each successively higher environment. For example, update Sandbox, then UAT, 
+then SIT, and finally Prod.
+
+4.	**Conduct smoke tests:** In each environment, perform smoke tests that 
+give you confidence that the upgrade was successful, prior to moving on to 
+the next environment. Since deployments can vary widely, it is up to you to 
+determine tests that might be specific to your deployment.
+
+5.	**Migrate during appropriate times:** Choose a time to migrate when 
+support is more likely to be available in case you encounter problems, such 
+as when you are most likely to be able to obtain support relatively promptly. 
+Prefer earlier in the week over later in the week (particularly avoiding 
+Fridays, if possible).
+
+## Updating ORCA Version
+
+To update your ORCA version:
+
+1.	Find the desired release on the [ORCA Releases page](https://github.com/nasa/cumulus-orca/releases).
+
+2.	Update the `source` in your Terraform deployment files for ORCA by 
+replacing `vx.x.x` with the desired version of ORCA.
+
+3.	Run `terraform init` to get the latest copies of your updated modules.
+
+## Update Data Persistence Resources

```

Comment:
Remove `## Update Data Persistence Resources` section. This is for Cumulus only

Reply:
done